sink/src/main/java/com/datastax/oss/kafka/sink/KafkaSchema.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.kafka.sink;
+
+import com.datastax.oss.common.sink.AbstractField;
+import com.datastax.oss.common.sink.AbstractSchema;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+
+/** Schema. */
+public class KafkaSchema implements AbstractSchema {
+
+  private static final KafkaSchema INT8 = new KafkaSchema(Schema.INT8_SCHEMA);
+
+  private static final KafkaSchema INT16 = new KafkaSchema(Schema.INT16_SCHEMA);
+  private static final KafkaSchema INT32 = new KafkaSchema(Schema.INT32_SCHEMA);
+  private static final KafkaSchema INT64 = new KafkaSchema(Schema.INT64_SCHEMA);
+  private static final KafkaSchema FLOAT32 = new KafkaSchema(Schema.FLOAT32_SCHEMA);
+  private static final KafkaSchema FLOAT64 = new KafkaSchema(Schema.FLOAT64_SCHEMA);
+  private static final KafkaSchema BOOLEAN = new KafkaSchema(Schema.BOOLEAN_SCHEMA);
+  private static final KafkaSchema STRING = new KafkaSchema(Schema.STRING_SCHEMA);
+  private static final KafkaSchema BYTES = new KafkaSchema(Schema.BYTES_SCHEMA);
+
+  public static KafkaSchema of(Schema schema) {
+    switch (schema.type()) {
+      case INT8:
+        return INT8;
+      case INT16:
+        return INT16;
+      case INT32:
+        return INT32;
+      case INT64:
+        return INT64;
+      case FLOAT32:
+        return FLOAT32;
+      case FLOAT64:
+        return FLOAT64;
+      case BOOLEAN:
+        return BOOLEAN;
+      case STRING:
+        return STRING;
+      case BYTES:
+        return BYTES;
+      default:
```

Comment:
shouldn't we handle `Collections` here as well?

Reply:
I didn't find constants for the other types in org.apache.kafka.connect.data.Schema, so probably we cannot do better